src/atoms/checkbox/component.tsx

```diff
@@ -131,14 +131,14 @@ const StyledCheckbox = styled.div<IStyledCheckboxProps>`
 interface ICheckBoxProps extends IComponentProps<ICheckboxTheme> {
   text: string;
   isChecked: boolean;
-  isDisabled?: boolean;
+  isEnabled?: boolean;
```

Comment:
I have made a change here so things are consistent throughout the project.

Reply:
ah can you please change things in the other direction - we want the defaults to be falsey

Reply:
but anyway we cannot change these variables or it would break existing uses, we should do this in a separate pr i think. it should come under this task: https://trello.com/c/VCka21XL/858-change-default-props-to-use-new-method

Reply:
I will revert the changes